staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1/doc.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=k8s.io/kubernetes/vendor/k8s.io/kube-aggregator/pkg/apis/apiregistration
+
+// API Registration is responsible for registering an API `Group`/`Version` with
+// another kubernetes like API server. The `APIService` holds information
+// about the other API server in `APIServiceSpec` type as well as general
```

Comment:
I was thinking a general idea of what these APIs are doing is helpful. I think I can sell this instead of one pager Brian asked for, what do you think?